pages/learn/manage/configuration.md

```diff
@@ -1,34 +1,16 @@
 ---
-title: Configuration variables
+title: Configuration
 ---
 
-# Configuration variables
+# Configuration
 
-Configuration variables allow you to provide runtime configuration to the host OS and supervisor. These variables all begin with `{{ $names.company.allCaps}}_` or `RESIN_`. Beginning with supervisor v7.0.0, a number of them appear automatically in your dashboard when your device is provisioned.
+Beyond [service variables][service-variables], you can also configure a lot of the device's behaviour with Supervisor and device specific variables.
 
-Configuration variables can be managed at both the fleet and device level.
+To see the complete list of variables available go to the [Configuration Variables][configuration-reference] reference. 
 
-__Note:__ Configuration variables defined in the dashboard will not apply to devices in [local mode][local-mode].
+**Note:** Some of these settings can potentially brick your device so we encourage that you read our [Advanced boot settings][boot-config-guide] guide.  
 
-## Variable list
```

Comment:
@20k-ultra We need to consider something important before removing any headings. We can't ignore the fact that users or even balenistas might have picked up the heading's anchor link and documented it somewhere else for the list of variables one can use. Deleting this heading won't break the page or go 404 but if some person clicks this link https://www.balena.io/docs/learn/manage/configuration/#variable-list then they won't be redirected to the right heading. We came across this on the fleet renamethon as we changed headings, we found out the hard way that anchor links can't be redirected without a bunch of hacks on the frontend side. Hence, anytime a heading needs to be deleted. It makes me uneasy. 

Example: https://www.balena.io/docs/learn/manage/actions/#rename-application 
  --- became -->      https://www.balena.io/docs/learn/manage/actions/#rename-fleet

Reply:
You can see the change in behavior of these 2 links.